app/screens/ServerView.tsx

```diff
@@ -204,7 +206,8 @@ const ServerView: React.FC<{
           placeholderTextColor="grey"
           selectionColor={colors.text.secondary}
           textContentType="username"
-          autoCompleteType="username"
+          autoComplete="username"
```

Comment:
It looks like you're trying to use newer properties here actually, we're still on react-native 0.64.1 right now which uses `autoCompleteType`: https://reactnative.dev/docs/0.64/textinput#autocompletetype-android

Reply:
Oh. I know nothing about React Native, I guess that’s why 🙂

Reply:
How about at least `autoCorrect=false` and `autoCapitalize=none`?

Reply:
You know I thought it was already not auto-capitalizing these fields based on the types currently set, but it looks like I was trying to use another hint here that actually had no effect on android (`textContentType`).  I think it would be best to add `autoCapitalize="none"` to all of them (address, username, password) now after a quick test here.

As for `autoCorrect`, I'd rather leave that on for username as I know the keyboard has a habit of remembering those anyway, and it already shouldn't be auto-correcting passwords, but I think it would be good to turn it on for the address at least.